File: HandwritenRecognition.Cross/DataTransferObjects/OcrJobDto.cs
namespace HandwritenRecognition.Cross.DataTransferObjects;

public class OcrJobDto
{
    public Guid JobId { get; set; }
    public OcrJobStatus Status { get; set; }
    public OcrResultDto? Result { get; set; }
    public string? Error { get; set; }
}

File: handwritten-recognition-client/src/model/dto/OcrJobDto.ts
import type OcrJobStatus from "../../cross/enums/OcrJobStatus"


export type OcrJobDto={
    jobId:object,
    error:string,
    status:OcrJobStatus,
    fileName:string,
    preprocessedFileName:string,
    createdAt:Date,
    updatedAt:Date
}